src/app/modules/feature-flags/feature-flags.service.ts

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose'
+import { okAsync, ResultAsync } from 'neverthrow'
+
+import { createLoggerWithLabel } from '../../config/logger'
+import getFeatureFlagModel from '../../models/feature_flag.server.model'
+import { DatabaseError } from '../core/core.errors'
+
+const logger = createLoggerWithLabel(module)
+const FeatureFlagModel = getFeatureFlagModel(mongoose)
+
+export const getEnabledFlags = (): ResultAsync<string[], DatabaseError> => {
```

Comment:
Please add a `TODO` that eventually we likely need a local caching system for feature flags, with some interval based refresh. That's because querying DB every time should be unnecessary for infrequently changing resources like feature flags.

You could check how we did it for OIDC remote NDI conf (cc @tshuli)
1) load on bootstrap (and fail to start if unable to fetch)
2) use local copy for all subsequent uses
3) set up a interval-based refresh, if failure to refesh: log but keep using stale data (no runtime or user visible errors)

I'm actually wondering if we need to have that caching system from day 1 for feature flags because I have a little worry that we are possibly the number of requests to DB quite a bit 🤔 

In my initial thought for a feature flag system, we'd just use env vars, meaning the feature flag value would be static for the entire lifetime of the process. Even if the feature flags come from DB now, doing a singular boostrap-time load is probably enough for now, and we can always do an env restart to force a refetch if needed.

Reply:
done in [06c7f79](https://github.com/opengovsg/FormSG/pull/6286/commits/06c7f792af2967f467d27eecdfd0abd66677da14)